Makefile

```diff
@@ -35,6 +35,28 @@ verify-generated-files:
 	@git diff --quiet || (echo 'Modifications exist locally! Run `make javascript` or `make sass` to update bundled files.'; exit 1)
 
 ################
+#Translations Handling
+################    
+# creates the django-partial.po & django-partial.mo files
+extract_translations:
+	i18n_tool extract -v
```

Comment:
Could you add `.tox` to the ignore directories specified here: https://github.com/edx/edx-ora2/blob/76acce9523b9a1d4b54294872ae45930aacc6f44/openassessment/locale/config.yaml#L76?

Reply:
whoa, new github link expandos